server/src/main/java/com/linecorp/centraldogma/server/metadata/ApplicationType.java

```diff
@@ -13,10 +13,21 @@
  * License for the specific language governing permissions and limitations
  * under the License.
  */
+
+package com.linecorp.centraldogma.server.metadata;
+
 /**
- * Central Dogma internal classes.
+ * The type of application.
  */
-@NonNullByDefault
-package com.linecorp.centraldogma.server.internal.command;
+public enum ApplicationType {
```

Comment:
I think these are the possible types that can authenticate applications.
```suggestion
public enum ApplicationAuthType {
```
or simply

```suggestion
public enum AppAuthType {
```

Reply:
The new naming looks good.